Sources/UIScreen.swift

```diff
@@ -10,16 +10,18 @@ import SDL
 import SDL_gpu
 import func Foundation.round
 
-
 extension SDLWindowFlags: OptionSet {}
 
 public extension UIScreen {
-    // Default value for testing. Gets set in TestSetup.swift
-    internal(set) public static var main: UIScreen!
+    internal(set) public static var main: UIScreen! {
+        didSet { CALayer.layerTreeIsDirty = true }
+    }
 }
 
 public final class UIScreen {
-    private var rawPointer: UnsafeMutablePointer<GPU_Target>!
+    // If we could use `private` members from an extension this would be private
+    // Keep that in mind when using it: i.e. if possible, don't ;)
+    internal var rawPointer: UnsafeMutablePointer<GPU_Target>!
```

Comment:
maybe this is a viable case for prefixing with `_`

Reply:
Maybe, but I noticed when prefixing `UIFont.availableFonts` that putting an underscore means these vars appear *first* in code completion, which is kind of the opposite of what we want. So I think for now it'd be best to just look out for this in code reviews

Reply:
Hi @mouselangelo, that could be an interesting option if you have an idea how we can avoid warnings popping up all through our code where we are using it legitimately? My impression is we'd need to have a lot of boilerplate at those places to suppress the warnings?

Reply:
Yup, I agree. Will add unnecessary boilerplate.